### PR TITLE
Add chameleon projector in thief toolbox

### PR DESCRIPTION
**Pull request by ERORR404V1**
## About the PR
Add chameleon projector in thief toolbox(chameleon kit)

## Why / Balance
More variability

## Technical details
None

## Media
- [] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added chameleon projector in thief toolbox!

### CODE REVIEW

**Comment by KingOnBudget** [comment]
in which kit did you out the chameleon projector??

File: Resources/Locale/en-US/thief/backpack.ftl [hunk 1]
```diff
@@ -15,9 +15,9 @@ thief-backpack-button-deselect = Select [X]
 
 thief-backpack-category-chameleon-name = chameleon's kit
 thief-backpack-category-chameleon-description =
-    Includes a full set of clothing that contain
-    chameleon technology, allowing you to disguise
-    as pretty much anything on the station.
+    Includes a full set of clothing that contains chameleon technology,
+    Contains a chameleon projector to help disguise yourself as objects,
```
Comment (Brandon-Huu) [inline Resources/Locale/en-US/thief/backpack.ftl:19]: Do you really have to directly mention the projector in the description?

Comment (ERORR404V1) [reply Resources/Locale/en-US/thief/backpack.ftl:19]: I don't think that's a problem

Comment (Brandon-Huu) [reply Resources/Locale/en-US/thief/backpack.ftl:19]: > how else are people meant to know about it

Then list every item in the kit, how else will they know

**Comment by TheShuEd** [comment]
i already did it in 
#25648

**Comment by ERORR404V1** [comment]
Should I close the PR?

**Comment by TheShuEd** [comment]
Hmmm
I think placing it in the chameleon set is a better idea than my option, so

**Pull request merged**